src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList/MedCentersList.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Button from '@components/common/Button/Button';
+import MedCenter from '@components/MedCentersPage/MedCentersList/MedCenter/MedCenter';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import { fetchMedCenters } from '@store/actionCreators/medCenters';
+import { MedCentersListWrapper, CenterWrapper } from './MedCentersListStyle';
+import { CreateNewOrEditLink } from '../MedCentersStyle';
+import { ROUTS } from '@constants/routs';
+import Error from '@components/common/Error/Error';
+
+interface IMedCentersListProps {
+  searchValue: string;
+}
+
+const MedCentersList: React.FC<IMedCentersListProps> = ({ searchValue }) => {
+  const dispatch = useDispatch();
+  const { medCenters, error } = useTypesSelector((state) => state.medCenter);
+
+  useEffect(() => {
+    dispatch(fetchMedCenters());
+  }, []);
+
+  return (
+    <MedCentersListWrapper>
+      {medCenters
+        .filter((medCenter) => medCenter.name.toLowerCase().includes(searchValue.toLowerCase()))
```

Comment:
extract to const pls

Reply:
src/components/AdminPanel/AdminMedCenters/MedCenters/MedCentersList/MedCentersList.tsx    added a check function

Reply:
src/components/AdminPanel/AdminMedCenters/MedCenters/MedCentersList/MedCentersList.tsx